src/patternfly/components/TreeView/examples/TreeView.md

```diff
@@ -112,54 +112,54 @@ beta: true
     {{/tree-view-list-item}}
     {{#> tree-view-list-item tree-view-list-item--IsExpandable="true"}}
-      {{> tree-view-node tree-view-node--IsToggle="true" tree-view-node--text="Cost management"}}
+      {{> tree-view-node tree-view-node--HasToggle="true" tree-view-node--text="Cost management"}}
     {{/tree-view-list-item}}
     {{#> tree-view-list-item tree-view-list-item--IsExpandable="true"}}
-      {{> tree-view-node tree-view-node--IsToggle="true" tree-view-node--text="Sources"}}
+      {{> tree-view-node tree-view-node--HasToggle="true" tree-view-node--text="Sources"}}
     {{/tree-view-list-item}}
     {{#> tree-view-list-item tree-view-list-item--IsExpandable="true"}}
-      {{> tree-view-node tree-view-node--IsToggle="true" tree-view-node--text="This is a really really really long folder name that overflows from the width of the container."}}
+      {{> tree-view-node tree-view-node--HasToggle="true" tree-view-node--text="This is a really really really long folder name that overflows from the width of the container."}}
     {{/tree-view-list-item}}
   {{/tree-view-list}}
 {{/tree-view}}
 ```
 
-### With-checkboxes
+### With checkboxes
 
 ```hbs
 {{#> tree-view}}
   {{#> tree-view-list tree-view-list--IsRoot="true"}}
     {{#> tree-view-list-item tree-view-list-item--IsExpandable="true" tree-view-list-item--IsExpanded="true"}}
-      {{> tree-view-node tree-view-node--IsToggle="true" tree-view-node--HasCheckbox="true" tree-view-node--text="Application launcher"}}
+      {{> tree-view-node tree-view-node--HasToggle="true" tree-view-node--type="div" tree-view-node--HasCheckbox="true" tree-view-node--IsChecked="true" tree-view-node--text="Application launcher"}}
       {{#> tree-view-list newcontext}}
         {{#> tree-view-list-item tree-view-list-item--IsExpandable="true" tree-view-list-item--IsExpanded="true"}}
-          {{> tree-view-node tree-view-node--IsToggle="true" tree-view-node--HasCheckbox="true" tree-view-node--IsActive="true" tree-view-node--text="Application 1"}}
+          {{> tree-view-node tree-view-node--HasToggle="true" tree-view-node--type="div" tree-view-node--HasCheckbox="true" tree-view-node--IsChecked="true" tree-view-node--text="Application 1"}}
           {{#> tree-view-list newcontext}}
             {{#> tree-view-list-item}}
-              {{> tree-view-node tree-view-node--HasCheckbox="true" tree-view-node--text="Settings"}}
+              {{> tree-view-node tree-view-node--type="div" tree-view-node--HasCheckbox="true" tree-view-node--IsChecked="true" tree-view-node--text="Settings"}}
             {{/tree-view-list-item}}
             {{#> tree-view-list-item}}
-              {{> tree-view-node tree-view-node--HasCheckbox="true" tree-view-node--modifier="pf-m-current" tree-view-node--text="Current"}}
+              {{> tree-view-node tree-view-node--type="div" tree-view-node--HasCheckbox="true" tree-view-node--IsChecked="true" tree-view-node--IsChecked="true" tree-view-node--text="Loader"}}
```

Comment:
```suggestion
              {{> tree-view-node tree-view-node--type="div" tree-view-node--HasCheckbox="true" tree-view-node-check--IsChecked="true" tree-view-node--text="Loader"}}
```

Reply:
I think you can just pass the `tree-view-node-check--IsChecked` through `{{> tree-view-node}}`.

Reply:
Nice spot! not sure why it was there twice